src/vm/arm/exceparm.cpp

```diff
@@ -54,6 +54,35 @@ FaultingExceptionFrame *GetFrameFromRedirectedStubStackFrame (T_DISPATCHER_CONTE
     return (FaultingExceptionFrame*)((TADDR)pDispatcherContext->ContextRecord->R4);
 }
 
+//Return TRUE if pContext->Pc is in DispatchStub
+BOOL IsDispatchStub(CONTEXT *pContext)
```

Comment:
I would also recommend to change the signature of this function to get the controlPc as a parameter instead of the whole context and renaming it to IsIPInDispatchStub (or IsIPInVirtualStub if we would handle resolve stubs too) to make it symmetrical to the IsIPInMarkedJitHelper.